<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QWidget" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="allUsersTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>571</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="queryButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>查找用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addUser">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>添加用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteUser">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>删除选中用户</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>62</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_auth1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>86</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>权限1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_auth2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>86</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>权限2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>62</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
